dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -0,0 +1,50 @@
+<LinkageCheckerFilter>
+  <!-- reactor-core's Traces catches Throwable to detect classes available in
+    Java 9+.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/816 -->
+  <LinkageError>
+    <Source>
+      <Package name="reactor.core.publisher.Traces" />
+    </Source>
+  </LinkageError>
+
+  <!-- GraalVM-related libraries depend on Java Compiler Interface (JVMCI) that
+    only exists in special JDK.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/929 -->
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="com.oracle.svm" />
+    </Source>
+  </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="com.oracle.graal" />
+    </Source>
+  </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="org.graalvm" />
+    </Source>
+  </LinkageError>
+
+  <!-- Mockito's MockMethodDispatcher uses special class loader to load
```

Comment:
The comment should explain why we don't care about this. In fact this doesn't need to be a comment. It can be another element, and include full HTML markup (if that's useful).

Reply:
Added explanation why the errors are false positives.

I just thought about adding `<reason>` element that would enable HTML markup. However it does not help users because the exclusion rules are for errors that won't be shown to users, nor in an HTML renderer. If you still think `<reason>` element helpful, feel free to file an issue. I'll check that later.

Reply:
Created https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1288

Reply:
I'm not OK with leaving this for later. This PR needs that information, and that information belongs in an element, not a comment.